tasty-plutus/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 This format is based on [Keep A Changelog](https://keepachangelog.com/en/1.0.0).
 
+## 5.3 -- 2022-01-17
+
+### Changes
+
+* Added `paysToPubKeyWithDatum`, `paysToWalletWithDatum` including a Datum for pub key outputs and
+  `spendsFromPubKeyWithDatum`, `spendsFromPubKeyWithDatumSigned`, `spendsFromWalletWithDatum`,
+  `spendsFromWalletWithDatumSigned` for pub key inputs
+
 ## 5.2 -- 2022-01-10
 
 ### Changes
```

Comment:
While we're at it, this should be ``### Changed``, not 'Changes'.

Reply:
Sorry, I admit I didn't read the keepachangelog documentation.